migrations/sqls/20201117091026-charge-versions-licence-id-constraints-up.sql

```diff
@@ -0,0 +1,13 @@
+/* make sure all licence_ids are populated */
+UPDATE water.charge_versions cv
+SET licence_id = l.licence_id FROM water.licences l where l.licence_ref = cv.licence_ref;
+
```

Comment:
If we're about to set a not null constraint here it may be safest to first delete any rows with nulls in this column